.env.test

```diff
@@ -0,0 +1,3 @@
+VITE_AIRTABLE_API_TOKEN=patuNpQmFvlO9sl9Z.ee29de109e3e595b4a7fbfaf8d9045f4f716ec369c7c7c98175cb9969873bac0
```

Comment:
nit: I'm not seeing this in the online lesson, but the good practice is to never commit .env files, since api tokens and other access keys can be exposed
the way to do this is to add the file name to the .gitignore